test/test_transforms_tensor.py

```diff
@@ -858,3 +858,33 @@ def test_gaussian_blur(device, channels, meth_kwargs):
         agg_method="max",
         tol=tol,
     )
+
+
+@pytest.mark.parametrize("device", cpu_and_gpu())
+@pytest.mark.parametrize(
+    "meth_kwargs",
+    [
+        {"fill": 1},
+        {"fill": 1.0},
+        {"fill": [1]},
+        {"fill": [1.0]},
+        {"fill": (1,)},
+        {"fill": (1.0,)},
+        {"fill": [1, 2, 3]},
+        {"fill": [1.0, 2.0, 3.0]},
+        {"fill": (1, 2, 3)},
+        {"fill": (1.0, 2.0, 3.0)},
+    ],
+)
+@pytest.mark.parametrize("channels", [1, 3])
+def test_elastic_transform(device, channels, meth_kwargs):
+    fill = meth_kwargs.get("fill")
+    if isinstance(fill, (list, tuple)) and len(fill) > 1 and channels == 1:
+        return
```

Comment:
xfail?

Reply:
I wouldn't, since this is just incompatible input data. This will never succeed and also shouldn't. IMO one should use `xfail` if something should work, but doesn't at the moment. Meaning, I only add `xfail` with the (sometimes naive) intention to remove it in the future again.